content/en/docs/started/installing-kubeflow.md

```diff
@@ -0,0 +1,97 @@
++++
+title = "Installing Kubeflow"
+description = "Overview of installation choices for Kubeflow"
+weight = 20
+
++++
+
+<a id="introduction"></a>
+## Introduction
+
+... short discussion about Kubeflow being a "Kubernetes" platform, with multiple components (not an "app"), 
+... link to kubeflow-overview for more details
+... (TODO) add a link to "Reference"/"Kubeflow Versioning Policies" in overview docs
+
+... short discussion about how Kubeflow is versioned, 
+... that is, components in Kubeflow can maintain their own versions, independent of the project-wide versions (like 1.2, 1.3)
+
+<a id="roll-your-own"></a>
+## Path 1: roll your own
+
+... disclaimer that any environment-specific issues are not supported by the Kubeflow community (use a distribution if you want support)
+
+... small discussion of the purpose/function of the kubeflow/manifests repo, 
+... that is, to mirror upstream manifests, and act as a central repository containing the "authoritative" manifests for each kubeflow release, to be used by distributions
+
+... link to kubeflow/manifests readme containing the "roll your own" instructions
+
+<a id="use-a-distribution"></a>
+## Path 2: use a distribution
+
```

Comment:
Consider adding a sentence or two here that describes what the user should do.

Reply:
@joeliedtke I have added some bullet points for what I would say.

What are your thoughts?